packages/cdktf-cli/test/ui/deploy.test.tsx

```diff
@@ -65,7 +65,7 @@ test("Apply", async () => {
   expect(stripAnsi(lastFrame())).toMatchInlineSnapshot(`
     "Deploying Stack: testing
     Resources
-     ⠋ foo.bar_352350
+     ⠋ FOO                  bar_352350          foo.bar_352350
```

Comment:
It didn't seem important to maintain this behavior.

Reply:
👍